jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudwatch/logs/editor/TableUtils.kt

```diff
@@ -74,6 +74,14 @@ class LogGroupTableSorter(model: ListTableModel<LogStream>) : TableRowSorter<Lis
     }
 }
 
+class LogGroupFilterTableSorter(model: ListTableModel<LogStream>) : TableRowSorter<ListTableModel<LogStream>>(model) {
```

Comment:
Why do we disable sort if its filtered?

Reply:
We don't, we change sort to column 0 because the search api is limited to sorting by name. Sort is disabled in both cases because otherwise loading more would make no sense.